pages/UI.php

```diff
@@ -384,31 +384,37 @@ function DisplayNavigatorGroupTab($oP)
 		
 		case 'details': // Details of an object
 			$sClass = utils::ReadParam('class', '');
-			$id = utils::ReadParam('id', '');
-			if ( empty($sClass) || empty($id))
-			{
-				throw new ApplicationException(Dict::Format('UI:Error:2ParametersMissing', 'class', 'id'));
+			if (empty($sClass)) {
+				throw new ApplicationException(Dict::Format('UI:Error:1ParametersMissing', 'class'));
 			}
 
-			if (is_numeric($id))
-			{
-				$oObj = MetaModel::GetObject($sClass, $id, false /* MustBeFound */);
-			}
-			else
-			{
-				$oObj = MetaModel::GetObjectByName($sClass, $id, false /* MustBeFound */);
+			$id = utils::ReadParam('id', null);
+			if (false === is_null($id)) {
+				if (is_numeric($id)) {
+					$oObj = MetaModel::GetObject($sClass, $id, false /* MustBeFound */);
+				} else {
+					$oObj = MetaModel::GetObjectByName($sClass, $id, false /* MustBeFound */);
+				}
+			} else {
+				$sAttCode = utils::ReadParam('attcode', '');
+				$sAttValue = utils::ReadParam('attvalue', '');
+
+				if (empty($sAttCode) || empty($sAttValue)) {
```

Comment:
```suggestion
				if ( (strlen($sAttCode) === 0) || (strlen(empty($sAttValue) === 0) ) {
```

We need to use `strlen()` otherwise accessing an object with $sAttValue at `"0"` will not work.

Reply:
Damn :( Thanks !
Fixed in 217e796c (your suggestion kept the empty() call for the second param)

Reply:
Woops, sorry! 😬